src/index.js

```diff
@@ -58,26 +58,34 @@ class ActiveStorageProvider extends React.Component<Props> {
     }
   }
 
+  _getHeaders = (): CustomHeaders => {
```

Comment:
I don't think we need this to be an arrow function, since we're never calling it from outside the class. And maybe let's use a proper getter, too: `get _headers() {`

Reply:
yeah you're probably right. I'll get that swapped out